CATesting/config.yaml

```diff
@@ -0,0 +1,119 @@
+# Constants
+{{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 10}}
+{{$qps := DefaultParam .qps 10}}
+# Adjust the following two for your cluster -
+# kubectl get node -o json | jq -r ".items[].status.allocatable.cpu"
+{{$NODE_FREE_CPU := DefaultParam .NODE_FREE_CPU 1}}
+# kubectl get node -o json | jq -r ".items[].status.allocatable.memory"
+# In MB
+{{$NODE_FREE_MEM := DefaultParam .NODE_FREE_MEM 3840}}
+
+# Variables
+{{$podsPerNamespace := MultiplyInt $PODS_PER_NODE .Nodes}}
+{{$totalPods := MultiplyInt $PODS_PER_NODE .Nodes}}
+# Pod Resource proportional to node to ensure consistent triggering of scale-up
+{{$POD_CPU_MILLI := DivideFloat $NODE_FREE_CPU (MultiplyFloat $PODS_PER_NODE .9)}}
+{{$POD_MEMORY_MB := DivideFloat $NODE_FREE_MEM (MultiplyFloat $PODS_PER_NODE .9)}}
+
+name: Simple_Bring_Up
+namespace:
+  number: 1
+tuningSets:
+- name: Uniformqps
+  qpsLoad:
+    qps: {{$qps}}
+
+steps:
+- name: Starting Measurements
+  measurements:
+  - Identifier: ScaleUpTimer
+    Method: Timer
+    Params:
+      action: start
+      label: scale_up_time
+  - Identifier: PodStartupLatency
+    Method: PodStartupLatency
+    Params:
+      action: start
+      labelSelector: group = nichotr
+      threshold: 1h
+  - Identifier: SchedulingThroughput
+    Method: SchedulingThroughput
+    Params:
+      action: start
+      labelSelector: group = nichotr
+
+- name: Creating pods
+  phases:
+    - namespaceRange:
+        min: 1
```

Comment:
Can you parameterize these namespaces? Does it have any defaults if this isn't provided?